test/helpers/env.js

```diff
@@ -203,6 +214,7 @@ env.IOS82 = env.DEVICE.match(/ios82/i);
 env.IOS83 = env.DEVICE.match(/ios83/i);
 env.IOS84 = env.DEVICE.match(/ios84/i);
 env.IOS9 = env.DEVICE.match(/ios9/i);
+env.IOS91 = env.DEVICE.match(/ios91/i);
```

Comment:
As you can see, there is `env.IOS8` and `env.IOS80` to distinguish the two, since there are things that iOS 8.0 does that 8.1, etc., do not. I don't know that that is the case here, just bringing it up to be on the lookout.